### DISCUSSION TITLE
Reading s3 bucket to create table from csv file

### DISCUSSION THREAD

**Discussion by jaarroyl** [Budibase/budibase#8406] [Help] [answered]
Hi guys! 

Im new on budibase and I would like to know how can I just read a csv file from Budibase using s3 integration.

I already connected my aws account to busibase, but due to Im not familiar with JavaScript I ask to you how to apply a transformer that allow me to read directly the csv file called "registro_evaluaciones.csv" to show in Retool.

![image](https://user-images.githubusercontent.com/41343847/198034759-adbb43b7-93ec-4de2-aaa5-10d8c8bae469.png)

best!!

**Comment by melohagan** [comment, marked as answer]
Hey @jaarroyl 

1. I have a PR out that will add this feature to the S3 connector if it gets merged: https://github.com/Budibase/budibase/pull/7521
2. You could try this in the meantime: https://github.com/Budibase/budibase/discussions/7439#discussioncomment-3484822

**Reply by jaarroyl** [reply]
Thanks a lot! Do you have a date for release this new feature?? Best,
